bowler/tool.py

```diff
@@ -101,6 +102,10 @@ def __init__(
             self.hunk_processor = hunk_processor
         else:
             self.hunk_processor = lambda f, h: True
+        if filename_matcher is not None:
+            self.filename_matcher = filename_matcher
+        else:
+            self.filename_matcher = lambda fn: fn.endswith(".py")
```

Comment:
Why not something like:

    self.filename_matcher = filename_matcher or filename_endswith(".py")